webapp/packages/plugin-data-spreadsheet-new/src/DataGrid/Formatters/CellFormatters/BooleanFormatter.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2021 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import { observer } from 'mobx-react-lite';
+import { useCallback, useContext } from 'react';
+import type { FormatterProps } from 'react-data-grid';
+import styled, { use, css } from 'reshadow';
+
+import { ResultSetFormatAction } from '@cloudbeaver/plugin-data-viewer';
+
+import { DataGridContext } from '../../DataGridContext';
+import { TableDataContext } from '../../TableDataContext';
+
+const styles = css`
+  value {
+    cursor: pointer;
+  }
+  value[|boolean] {
+    font-family: monospace;
+    white-space: pre;
+    line-height: 1;
+    vertical-align: text-top;
+  }
+`;
+
+function getClasses(rawValue: any) {
+  const classes = [];
+  if (rawValue === null) {
+    classes.push('cell-null');
+  }
+  return classes.join(' ');
+}
+
+export const BooleanFormatter: React.FC<FormatterProps> = observer(function BooleanFormatter({ column, row, rowIdx }) {
+  const context = useContext(DataGridContext);
+  const tableDataContext = useContext(TableDataContext);
+  const formatter = context?.model.source.getAction(context.resultIndex, ResultSetFormatAction);
+  const resultColumn = tableDataContext?.getColumnInfo(column.key);
+  const rawValue = formatter?.get(row[column.key]) ?? row[column.key];
+  const stringifiedValue = formatter?.toDisplayString(rawValue) ?? String(rawValue);
+  const value = rawValue === null ? stringifiedValue : `[${rawValue ? 'v' : ' '}]`;
+  const classes = getClasses(rawValue);
+
+  const getNextValue = useCallback((prev: boolean | null) => {
+    if (!resultColumn?.required && prev === false) {
+      return null;
+    }
+
+    return !prev;
+  }, [resultColumn]);
+
+  return styled(styles)(
+    <boolean-formatter as='div' className={`boolean-formatter ${classes}`} onDoubleClick={e => e.stopPropagation()}>
```

Comment:
`boolean-formatter` class is not used, also seems like `value` component can be removed and all props can be moved to `boolean-formatter`